### PR TITLE
chore: Bump deps

### PR DESCRIPTION
**Pull request by rschristian**
**What kind of change does this PR introduce?**

Chore

**Did you add tests for your changes?**

Existing cover and were updated

**Summary**

Bumps deps. More lock file line additions than removals is a bit disappointing. Was hoping (maybe foolishly) the opposite would happen.

Nothing user-facing saw a major bump.

**Does this PR introduce a breaking change?**

No

### CODE REVIEW

File: packages/cli/tests/images/build.js [hunk 1]
```diff
@@ -11,33 +11,32 @@ exports.default = Object.assign({}, common, {
...
+	'route-profile.chunk.7bcae.css': 61,
+	'route-profile.chunk.6d480.js': 3514,
+	'route-profile.chunk.6d480.js.map': 15454,
 });
 exports['default-esm'] = Object.assign({}, exports.default, {
 	'bundle.*.esm.js': 21135,
 	'bundle.*.esm.js.map': 111771,
 	'polyfills.*.esm.js': 5721,
 	'polyfills.*.esm.js.map': 21633,
-	'route-home.chunk.3cec8.js.map': 483,
```
Comment (rschristian) [inline packages/cli/tests/images/build.js:40]: Silly copy/paste error that snuck in.

**Comment by rschristian** [comment]
> pnp-test@workspace:. provides eslint (p77961) with version 6.8.0, which doesn't satisfy what eslint-config-preact and some of its descendants request

PNP is the most needlessly frustrating tool designed in recent years.

I'm going to submit a second PR to remove that CI test. It serves no value blocking merges until we go fix non-issues upstream.

**Review by developit — APPROVED** [review]

File: packages/cli/tests/images/build.js (hunk 1 above)
Comment (developit) [reply packages/cli/tests/images/build.js:40]: should be a wildcard?

Comment (rschristian) [reply packages/cli/tests/images/build.js:40]: The next line is the (correct) wildcard match:

https://github.com/preactjs/preact-cli/blob/49bba048d3c33ab490b45009957d27f8014848c5/packages/cli/tests/images/build.js#L40-L41

This was a non-esm entry that somehow snuck into the esm file list, only jumped out as the file hash changed

**Comment by merceyz** [comment]
> > pnp-test@workspace:. provides eslint (p77961) with version 6.8.0, which doesn't satisfy what eslint-config-preact and some of its descendants request
> 
> _sigh_ CI failure due to Yarn PNP not being provided some specific (but usable) version of ESLint.

The logs have expired by now but the message you quoted is a warning and doesn't impact the runtime.

> It serves no purpose besides being a canary that the 5 Yarn PNP users might have issues.

Ouch.

**Comment by rschristian** [comment]
> The logs have expired by now but the message you quoted is a warning and doesn't impact the runtime.

It was the CI that you had set up, which treated that as an error and would fail the job.

>> It serves no purpose besides being a canary that the 5 Yarn PNP users might have issues.
>
> Ouch.

I was referring to our users (of which I counted ~5), rather than Yarn PNP users in general, who were spotting the Yarn PNP issues.

I apologize for not being the kindest, but Yarn 2+, PNP included, has been some of the most frustrating software I've ever had to work around and this PR came after a long day of dealing with its issues.

I'll delete that comment though, it's inappropriate to keep around.

**Pull request merged**